pages/marina.vue

```diff
@@ -1,142 +1,140 @@
 <template>
-  <div>
-    <div>
-      <input type="range" min="1" max="500" value="100" class="slider">
-    </div>
-    
+<div>
+  <div class='login'>
```

Comment:
Qual sua ideia com esse login aqui

Reply:
Login
senha
pass

qual sua ideia?

Reply:
To aprendendo como que faz....
Caso a gente realmente tente fazer um petshop online, vai ser necessário que a pessoa faça login para poder concluir uma compra.
Mas por enquanto sou só eu brincando com o código.

Reply:
okay

Reply:
no mundo real vc tem que justificar tudo que você está criando, mas aqui é aprendizado 👍🏻